recipes-tag/pymongo/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "3.6.1" %}
```

Comment:
we don't need pymongo, it is the deaults channel .

Reply:
unless we need better than 3.4.0

Reply:
unfortunately, it's set by the developer [here](https://github.com/AustralianSynchrotron/lightflow/blob/master/setup.py#L47).
We need pymongo 3.6.0

I can ask to see if he can downgrade this requirement if adding this is a problem. It's very likely he doesn't need 3.6.

Reply:
fair enough.